web/src/components/tasks/RaffleBingoPaginator.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { RaffleBingoPaginator } from "@/components/tasks/RaffleBingoPaginator";
+import { Task } from "@/lib/types/types";
+import { getTaskStatusUpdatedMessage } from "@/lib/utils/helpers";
+import { generateTask } from "@/test/factories";
+import { useMockRouter } from "@/test/mock/mockRouter";
+import { useMockRoadmap } from "@/test/mock/mockUseRoadmap";
+import { setupStatefulUserDataContext, WithStatefulUserData } from "@/test/mock/withStatefulUserData";
+import { generateUserData } from "@businessnjgovnavigator/shared/test";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+jest.mock("next/router", () => ({ useRouter: jest.fn() }));
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+jest.mock("@/lib/roadmap/buildUserRoadmap", () => ({ buildUserRoadmap: jest.fn() }));
+
+jest.mock("@/lib/async-content-fetchers/fetchTaskByFilename", () => ({
+  fetchTaskByFilename: jest.fn(() =>
+    Promise.resolve({
+      default: "Mocked Task Content",
+    })
+  ),
+}));
+
+describe("<RaffleBingoPaginator />", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+    useMockRouter({});
+    setupStatefulUserDataContext();
+  });
+
+  const renderPaginator = (task: Task): void => {
+    render(
+      <WithStatefulUserData initialUserData={generateUserData({})}>
+        <RaffleBingoPaginator task={task} />
+      </WithStatefulUserData>
+    );
+  };
+
+  it("renders first page without clicking stepper", async () => {
+    const task = generateTask({});
+
+    renderPaginator(task);
+    await waitFor(() => {
+      expect(screen.getByTestId("stepper-0")).toBeInTheDocument();
+    });
+  });
+
+  it("renders NJ Municipality page", async () => {
```

Comment:
[pebble] I think there's a better name for this test. The goal of this test is to capture the state of task when you're viewing the second step. Consider renaming this to better reflect that.